src/main/java/org/owasp/esapi/reference/Log4JLogger.java

```diff
@@ -1,527 +0,0 @@
-
-	public Log4JLogger(String name) {
-		super(name);
-	}
-
-	/**
-	 * This method overrides {@link Logger#getInstance} by supplying
-	 * its own factory type as a parameter.
-	 */
-	public static Category getInstance(String name) {
-		return LogManager.getLogger(name, factory);
-	}
-
-	/**
-	 * This method overrides {@link Logger#getInstance} by supplying
-	 * its own factory type as a parameter.
-	 */
-	public static Category getInstance(Class clazz) {
-		return LogManager.getLogger(clazz.getName(), factory);
-	}
-
-	/**
-	 * This method overrides {@link Logger#getLogger} by supplying
-	 * its own factory type as a parameter.
-	 */
-	public static Logger getLogger(String name) {
-		return LogManager.getLogger(name, factory);
-	}
-
-	/**
-	 * This method overrides {@link Logger#getLogger} by supplying
-	 * its own factory type as a parameter.
-	 */
-	public static org.apache.log4j.Logger getLogger(Class clazz) {
-		return LogManager.getLogger(clazz.getName(), factory);
-	}
-
-	/**
-	 * {@inheritDoc}
-	 * Note: In this implementation, this change is not persistent,
-	 * meaning that if the application is restarted, the log level will revert to the level defined in the
-	 * ESAPI SecurityConfiguration properties file.
-	 */
-	public void setLevel(int level) {
-		try {
-			super.setLevel(convertESAPILeveltoLoggerLevel(level));
-		} catch (IllegalArgumentException e) {
-			this.error(org.owasp.esapi.Logger.SECURITY_FAILURE, "", e);
-		}
-	}
-	
-	/**
-	 * {@inheritDoc}
-	 * Explanation: Since this class extends Log4j's Logger class which has a
-	 * {@code getLevel()} method that returns {@code extended by org.apache.log4j.Level},
-	 * we can't simply have a {@code getLevel()} that simply returns an {@code int}.
-	 * Hence we renamed it to {@code getESAPILevel()}.
-	 */
-	public int getESAPILevel() {
-		Level level = super.getLevel();
-    return (level == null ) ? Level.OFF_INT : level.toInt();
-	}
-
-	/**
-	 * Converts the ESAPI logging level (a number) into the levels used by Java's logger.
-	 * @param level The ESAPI to convert.
-	 * @return The Log4J logging Level that is equivalent.
-	 * @throws IllegalArgumentException if the supplied ESAPI level doesn't make a level that is currently defined.
-	 */
-	private static Level convertESAPILeveltoLoggerLevel(int level) {
-		switch (level) {
-		case org.owasp.esapi.Logger.OFF:
-			return Level.OFF;
-		case org.owasp.esapi.Logger.FATAL:
-			return Level.FATAL;
-		case org.owasp.esapi.Logger.ERROR:
-			return Level.ERROR;
-		case org.owasp.esapi.Logger.WARNING:
-			return Level.WARN;
-		case org.owasp.esapi.Logger.INFO:
-			return Level.INFO;
-		case org.owasp.esapi.Logger.DEBUG:
-			return Level.DEBUG; //fine
-		case org.owasp.esapi.Logger.TRACE:
-			return Level.TRACE; //finest
-		case org.owasp.esapi.Logger.ALL:
-			return Level.ALL;
-		default: {
-			throw new IllegalArgumentException("Invalid logging level. Value was: " + level);
-		}
```

Comment:
As a note, these values were used to create the mapping in the Log4J log factory.  The intention is that the mapping is the same.